scheduler/Dockerfile

```diff
@@ -35,11 +35,12 @@ WORKDIR /opt/cook
 ## with fetched dependencies as long as project.clj isn't modified
 RUN lein deps
 
-# Copy the whole scheduler into the container
-COPY . /opt/cook/
-
 # Datomic setup
+COPY datomic /opt/cook/datomic
 RUN unzip -uo /opt/cook/datomic/datomic-free-0.9.5394.zip
```

Comment:
If we converted `datomic-free-0.9.5394.zip` to `datomic-free-0.9.5394.tar.gz`, then we could take advantage of the `ADD` command's ability to auto-extract tarballs.